src/ServiceControl.Plugin.Nsb5.SagaAudit/Messages/SagaChangeInitiator.cs

```diff
@@ -7,7 +7,7 @@ class SagaChangeInitiator
         public string InitiatingMessageId { get; set; }
         public string MessageType { get; set; }
         public bool IsSagaTimeoutMessage { get; set; }
-        public DateTime TimeSent { get; set; }
+        public DateTime? TimeSent { get; set; }
```

Comment:
@SimonCropp - In SI, not sure at the moment, I was going to check with @distantcam, in SP I think @gbiellem has fixed it. I'll double check. If not what's your proposal. To roll out SP and SI first enabling for decent display, before we roll out this change? Or do you have anything else in mind?